README

```diff
@@ -13,11 +13,15 @@ the library is working correctly.  Another example is given in the file
 test/minigzip.c.  The compression library itself is composed of all source
 files in the root directory.
 
-To compile all files and run the test program, follow the instructions given at
-the top of Makefile.in.  In short "./configure; make test", and if that goes
-well, "make install" should work for most flavors of Unix.  For Windows, use
-one of the special makefiles in win32/ or contrib/vstudio/ .  For VMS, use
-make_vms.com.
+To compile all files and run the test program, follow this instructions:
```

Comment:
Your example commands also install zlib.

Reply:
but not as expected:
```bash
bash-5.2$ cmake --install build --prefix /tmp/zlib
-- Install configuration: "Release"
-- Installing: /usr/local/lib/cmake/zlib/zlibConfigVersion.cmake
-- Up-to-date: /usr/local/lib/libz.a
-- Installing: /usr/local/lib/cmake/zlib/zlibTargets.cmake
-- Installing: /usr/local/lib/cmake/zlib/zlibTargets-release.cmake
-- Up-to-date: /usr/local/include/zconf.h
-- Up-to-date: /usr/local/include/zlib.h
-- Up-to-date: /usr/local/share/man/man3/zlib.3
-- Up-to-date: /usr/local/share/pkgconfig/zlib.pc
bash-5.2$ 
```